python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -273,10 +274,23 @@ def makePlot(
                 values = values[goodValues]  # As a precaution.
 
                 # Make a concise human-readable label for the plot.
+                plotInfo["unit"] = "N/A"  # Unless overridden.
+                if hasattr(mapData, "metadata") and all(
+                    key in mapData.metadata for key in ["DESCRIPTION", "OPERATION", "UNIT"]
+                ):
+                    metadata = mapData.metadata
+                    plotInfo["description"] = metadata["DESCRIPTION"]
+                    plotInfo["operation"] = metadata["OPERATION"]
+                    if metadata["UNIT"]:
+                        plotInfo["unit"] = metadata["UNIT"]
```

Comment:
I think the logic here is not quite right.  If `metadata["UNIT"]` is set to `""` (empty string) then the unit really is `dimensionless` and not `N/A`.  So I think that (a) here should set `plotInfo["unit"] = "dimensionless"` if there's no unit; (b) below it should check for either `N/A` or `dimensionless` and not put the unit on the plot.  And that will create the right distinction in the plot title.

Reply:
Done!